wp-content/themes/oceanwp-child/assets/css/template-styles.css

```diff
@@ -219,7 +219,7 @@ hr{
 /* Blog Styles Start */
 .blog-subheader{
     height:300px;
-    background: linear-gradient(0deg, rgba(255 255 255 / 80%), rgba(255 255 255 / 80%)), url(http://localhost/ggrc_website/wp-content/uploads/2022/01/Arctic-Terns-Steven-Calcote-scaled.jpg);
+    background: linear-gradient(0deg, rgba(255 255 255 / 80%), rgba(255 255 255 / 80%)), url(../../../../uploads/2022/01/Arctic-Terns-Steven-Calcote-scaled.jpg);
```

Comment:
Do we really need all of the `..`? I think it should work with just `/wp-content/...`

Reply:
I tried with `/wp-content/` but didn't work, the url it picks is like this `http://localhost/wp-content/...`